docs/kyma-environment-broker/03-15-trial-cleanup-cronjob.md

```diff
@@ -0,0 +1,45 @@
+# Trial Cleanup Job
+
+Trial Cleanup Job is a Job that makes the SKR instances with the trial plan expire if those are 14 days old.
+Expiration means that the SKR instance is suspended and the `expired` flag is set.
+
+## Details
+
+For each instance meeting criteria, a PATCH request is sent to Kyma Environment Broker (KEB). This instance is marked as `expired`, and if it is in the `succeeded` state, the suspension process is started. 
+If the instance is already in the `suspended` state, this instance is only marked as `expired`. 
+
+### Dry-run mode
+If you need to test your the Job, you can run it in the dry-run mode.
+In that mode, the Job only logs the information about the candidate instances (i.e. instances meeting the configured criteria). No changes are made.
+
+## Prerequisites
+
+The Trial Cleanup Job requires access to:
+- Database, to get the IDs of the instances with `trial` plan which are not expired yet. 
+- KEB to initiate the SKR instance suspension.
+
+## Configuration
+
+The Job is a CronJob with schedule that can be configured as the parameter in `management-plane-config`.
+By default, the CronJob is set to run every day at 1:15 am:
+```yaml  
+kyma-environment-broker.trialCleanup.schedule: "15 1 * * *"
+```
+
+Use the following environment variables to configure the Job:
+
+| Environment variable | Description                                                                                                                    | Default value                            |
+|---|--------------------------------------------------------------------------------------------------------------------------------|------------------------------------------|
+| **APP_DRY_RUN** | Specifies whether to run in the [`dry-run` mode](#details). dry run only logs candidate instances                              | `true`                                   |
```

Comment:
```suggestion
| **APP_DRY_RUN** | Specifies whether to run in the [`dry-run` mode](#details).                                                                    | `true`                                   |
```

You explain the mode in the section you link to. Let's not duplicate content :)

Reply:
Sure.

Reply:
It's still there :D

Reply:
It should not.